controls/radchartview/features/plot-area.md

```diff
@@ -51,6 +51,42 @@ You can customize the plot area visual via the __PlotAreaStyle__ property of the
 #### __Figure 3: PlotAreaStyle example__
 ![PlotAreaStyle example](images/radchartview-features-plot-area-2.png)
 
+## Displaying stacked elements 
+
+By default the order in which the elements are defined in XAML determines which will be in front of the other when displayed. You can control which element is displayed on top by setting the elements' **ZIndex** property. RadChartView elements which have this property are [Series]({%slug radchartview-series-chartseries%}), [Annotations]({%slug radchartview-annotations-overview%}) and [Axes]({%slug radchartview-axes-axis%}).
```

Comment:
"By default the order in which the elements are defined in XAML determines" - this is not true. Actually, the elements could be defined in code. Here we could use something like "The stack order of the elements in the chart is determined by their positions in the items collection that holds them." and then you can say "You can additionally control the stack order by setting the ZIndex property of the corresponding chart element."